.gitignore

```diff
@@ -1,3 +1,29 @@
 .idea
 .venv
 __pycache__
+
```

Comment:
I really don't think we need a tenth of these.

Reply:
I'll try and narrow it down to only the ones we need